src/dds/rs-dds-sensor-proxy.cpp

```diff
@@ -673,5 +678,25 @@ void dds_sensor_proxy::add_processing_block( std::string const & filter_name )
     }
 }
 
+void dds_sensor_proxy::add_processing_block_settings( const std::string & filter_name,
+                                                      std::shared_ptr< librealsense::processing_block_interface > & ppb ) const
+{
+    if( rsutils::string::nocase_equal( filter_name, "Decimation Filter" ) )
+        if( !ppb->supports_option( RS2_OPTION_STREAM_FILTER ) )
+            LOG_ERROR( "Decimation Filter does not support stream filter option" );
+        else
+            if( rsutils::string::nocase_equal( get_name(), "RGB Camera" ) )
+            {
+
```

Comment:
Please consider to remove this blank line

Reply:
Removing

Reply:
Removing